src/main.rs

```diff
@@ -246,17 +225,59 @@ fn migrate(data_string: String) -> Result<String, anyhow::Error> {
     Ok(output_path_str.to_string())
 }
 
-#[launch]
-fn rocket() -> Rocket<Build> {
+async fn rm_file_after_expiration(database: &Arc<Mutex<Connection>>) -> Result<(), anyhow::Error> {
+    let database = database.lock().await;
+    let mut stmt = database.prepare(format!("SELECT * FROM {}", TABLE_NAME).as_str())?;
+    let rows = stmt.query([])?;
+    let rows = rows.mapped(|row| {
+        Ok(Entry {
+            uuid: row.get(0)?,
+            file_path: row.get(1)?,
+            creation_time: row.get(2)?,
+        })
+    });
+    for row in rows {
+        let row = row?;
```

Comment:
```suggestion
    })
    .collect::<Result<Vec<Entry>, rusqlite::Error>>()?;
    for row in rows {
```
This moves the error reporting out of the loop. I am a bit torn if it is really that much better, but in my opinion it at least seems like a more 'rusty' way to do this.